src/server/twemcache-rs/src/main.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2021 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+#[macro_use]
+extern crate rustcommon_logger;
+
+use backtrace::Backtrace;
+use std::sync::Arc;
+
+use config::TwemcacheConfig;
+use pelikan_twemcache_rs::TwemcacheBuilder;
+
+use rustcommon_logger::Logger;
+
+fn main() {
+    // custom panic hook to terminate whole process after unwinding
+    std::panic::set_hook(Box::new(|s| {
+        error!("{}", s);
+        println!("{:?}", Backtrace::new());
+        std::process::exit(101);
+    }));
+
+    // load config from file
+    let config = if let Some(file) = std::env::args().nth(1) {
```

Comment:
In rpc-perf you have the ability to override config in command line. Can we do something similar here?

Reply:
I'd urge that we're careful about how much we want to allow for that. I'm okay if we want to use `clap` to do something a little more advanced here. With rpc-perf and rezolus, we specify the config with `--config [FILE]` instead of just taking a bare argument at a fixed position - if you're okay doing something similar here, it's easy to add.

We should just be careful about how much we want to allow config overrides on the CLI - rpc-perf is a somewhat extreme example with lots of overriding possible - rezolus is a lot more reserved, only taking verbosity and config file on the CLI.

What would you want to allow for pelikan backends?